packages/oui/_settings.less

```diff
@@ -1,5 +1,5 @@
 // Settings - Global
-@oui-version: "2.3.0";
+@oui-version: "2.4.1";
```

Comment:
Maybe we could go with ``@oui-version: `Math.ceil(Math.random() * 100000)`;``

Reply:
I'll patch for now but we will apply a permanent fix